onyo/lib/onyo.py

```diff
@@ -120,15 +120,15 @@ def set_config(self,
 
     def get_config(self,
                    name: str) -> str | None:
-        """Get effective value of config `name`.
+        r"""Get effective value of config `name`.
 
         This is considering regular git-config locations and checks
         `OnyoRepo.ONYO_CONFIG` as fallback.
         """
         return self.git.get_config(name) or self.git.get_config(name, self.git.root / self.ONYO_CONFIG)
 
     def get_asset_name_keys(self) -> list[str]:
-        """Get a list of keys required for generating asset names
+        r"""Get a list of keys required for generating asset names
```

Comment:
Trailing `.` missing.

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.